README.md

```diff
@@ -28,31 +28,159 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## Authentication
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+Planet's APIs require an account for use.
+[Sign up here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+Authentication information can be obtained and stored locally with the CLI
+`planet init` command. Both the Python library and CLI will default to using
+the stored authentication information, if available (not yet implemented, see
+issues #244 and #248).
 
+Additionally, authentication information can be provided to the CLI by
+storing the api key in the environmental variable `PL_API_KEY` or by specifying
+the api key with the flag `-k` or `--api-key` (TODO: link to cli documentation
+here).
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library Usage
 
+The client modules within the Python library are asynchronous, which greatly
+speeds up many interactions with Planet's APIs. Support for asynchronous
+development is native to Python 3.6+ via the
+[`asyncio` module](https://docs.python.org/3/library/asyncio.html) and is
+fairly easy to implement.
```

Comment:
"fairly easy to implement" could come across as a little harsh, especially for our users that are new to the space or who may be struggling to grasp these concepts initially. I think we want them to feel encouraged and empowered instead. What do you think about saying something like "Support for asynchronous development is native to Python 3.6+ via the [`asyncio` module](...). For the sake of brevity, here is a starting point if you feel unfamiliar or would like some more information about asynchronous programming in Python to help get you started: https://project-awesome.org/timofurrer/awesome-asyncio."?

Reply:
Yeah, good catch. We can be more understanding and supportive here. Updated the wording here to include this awesome (hah!) resource.